fladle-plugin/src/test/java/com/osacky/flank/gradle/integration/FlankGradlePluginIntegrationTest.kt

```diff
@@ -126,6 +127,69 @@ class FlankGradlePluginIntegrationTest {
     assertThat(result.output).contains("debugApk must be specified")
   }
 
+  fun setUpDependOnAssemble(dependsOnAssemble: Boolean): BuildResult  {
+      writeBuildGradle(
+          """plugins {
+          id "com.osacky.fladle"
+          id "com.android.application"
+         }
+         repositories {
+              google()
+              mavenCentral()
+          }
+         android {
+             compileSdkVersion 29
+             defaultConfig {
+                 applicationId "com.osacky.flank.gradle.sample"
+                 minSdkVersion 23
+                 targetSdkVersion 29
+                 versionCode 1
+                 versionName "1.0"
+                 testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+             }
+             testOptions {
+                 execution 'ANDROIDX_TEST_ORCHESTRATOR'
+              }
+         }
+         fladle {
+           serviceAccountCredentials = project.layout.projectDirectory.file("foo")
+           dependOnAssemble = $dependsOnAssemble
+         }
+    """.trimIndent()
+      )
+      testProjectRoot.newFile("foo").writeText("{}")
+      testProjectRoot.newFolder("src/main")
+      testProjectRoot.newFile("src/main/AndroidManifest.xml").writeText("""
+          <?xml version="1.0" encoding="utf-8"?>
+          <manifest package="com.osacky.flank.gradle.sample" xmlns:android="http://schemas.android.com/apk/res/android" />
+      """.trimIndent())
+      return GradleRunner.create()
+          .withProjectDir(testProjectRoot.root)
+          .withPluginClasspath()
+          .withGradleVersion("6.1.1")
+          .withArguments("runFlank")
+          .buildAndFail()
```

Comment:
can we verify that the error message is exactly what we expect it to be here? there could be other reasons for the build failure here and we just want to make sure it is the same one as what we expected here.